_posts/2021-09-18-p2pool-released.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "Monero P2Pool is now live"
+summary: P2Pool centralized pool for Monero mining is live on the mainnet
+tags: [community]
+author: QuickBASIC
+---
+
+We are happy to announce that Monero P2Pool is live on the mainnet and binaries have been [released](https://github.com/SChernykh/p2pool/releases).

```

Comment:
Just to change from "mainnet" that is used multiple times. This is clearer for less technical people who don't know what mainnet is.
```suggestion
We are happy to announce that Monero P2Pool is live on the Monero network and binaries have been [released](https://github.com/SChernykh/p2pool/releases).

```

Reply:
Ack.